flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/SqlClient.java

```diff
@@ -213,6 +215,20 @@ protected static void startClient(String[] args, Supplier<Terminal> terminalFact
         }
     }
 
+    private static void checkArgs(String[] args) {
+        for (String arg : args) {
+            checkFilePath(arg);
```

Comment:
Why every arg needs to check path.. Only the values of the optoin `-i` and `-f` need check.

Reply:
yes ,I closed the previous PR, and I will fix it. only check -i -f.